static/js/components/PersonalTab.js

```diff
@@ -25,19 +26,27 @@ class PersonalTab extends React.Component {
   updateGender(gender) {
     const { dispatch, profile: { profileCopy } } = this.props;
     let clone = Object.assign({}, profileCopy, {
-      gender: gender
+      gender: gender ? gender.value : null
     });
     dispatch(updateProfile(clone));
   }
 
   updateLanguage(language) {
     const { dispatch, profile: { profileCopy } } = this.props;
     let clone = Object.assign({}, profileCopy, {
-      'preferred_language': language
+      'preferred_language': language ? language.value : null
     });
     dispatch(updateProfile(clone));
   }
 
+  saveAndContinue() {
+    const { dispatch, profile: { profileCopy } } = this.props;
+
+    dispatch(saveProfile(SETTINGS.username, profileCopy)).then(() => {
+      // TODO: update to next tab
```

Comment:
That makes sense. Any thoughts @annagav? I'm going to avoid the work for this PR since there's only one tab